src/utils/proxy.ts

```diff
@@ -859,15 +859,13 @@ const onRequest = async (
   }
 
   const maybeNotifyActivity = () => {
-    if (req.method === 'GET' && api && process.env.NETLIFY_DEV_SERVER_ID) {
+    if (api && process.env.NETLIFY_DEV_SERVER_ID && (req.method === 'GET' || req.url?.includes('/.ntlfy-dev/'))) {
```

Comment:
Is `/.ntlfy-dev/` at the beginning of the URL path? If so, so we check for that specifically as opposed to allowing it anywhere on the URL?